codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -107,20 +239,85 @@ protected Result beginEncode(HttpResponse headers, CharSequence acceptEncoding)
 
         String targetContentEncoding;
         switch (wrapper) {
-        case GZIP:
-            targetContentEncoding = "gzip";
-            break;
-        case ZLIB:
-            targetContentEncoding = "deflate";
-            break;
-        default:
-            throw new Error();
-        }
-
-        return new Result(
-                targetContentEncoding,
-                new EmbeddedChannel(ZlibCodecFactory.newZlibEncoder(
-                        wrapper, compressionLevel, windowBits, memLevel)));
+            case GZIP:
+                targetContentEncoding = "gzip";
+                break;
+            case ZLIB:
+                targetContentEncoding = "deflate";
+                break;
+            default:
+                throw new Error();
+        }
+
+        return new Result(targetContentEncoding,
+                          new EmbeddedChannel(ZlibCodecFactory.newZlibEncoder(wrapper,
+                                                                              compressionLevel,
+                                                                              windowBits,
+                                                                              memLevel)));
```

Comment:
In this case, it might look better to break the line before the first parameter.